packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/choice/choice.controller.js

```diff
@@ -64,6 +72,14 @@ export default class TelephonyLinePhoneOrderChoiceCtrl {
       });
   }
 
+  filterPhoneForMgcp(phones) {
+    return phones.filter((phone) =>
```

Comment:
Hello! Do you know if "phones" is always !== null ? Could throw an error if you an empty phone list?